perf/terraform/.gitignore

```diff
@@ -1,5 +1,6 @@
 # Local .terraform directories
 **/.terraform/*
+configs/local/.terraform.lock.hcl
```

Comment:
The lock file's purpose is to ensure that we have consistent `terraform` executions across machines. To ensure that, one needs to check it into version control. Is that not the best practice for terraform lock files?

Reply:
Terraform automatically creates or updates the dependency lock file each time [the terraform init command](https://developer.hashicorp.com/terraform/cli/commands/init) is run and whilst it is recommended to version it, it is for the use case of discussing potential changes to external dependencies - which would be dictated by the [remote terraform configuration](https://github.com/libp2p/test-plans/blob/master/perf/terraform/configs/remote/.terraform.lock.hcl) as opposed one's changes on their local machine, so versioning the local configuration lock file just makes the PR more noisy imo.

That being said this isn't a strong opinion if you feel there is a benefit in reviewing these changes.